src/remotestorage.js

```diff
@@ -798,12 +798,6 @@ Object.defineProperty(RemoteStorage.prototype, 'caching', {
 });
 
 /*
- * @property remote
- *
- * Access to the remote backend used. Usually a <RemoteStorage.WireClient>.
```

Comment:
Has this property gone, or is it just `@private`?

Reply:
This was just a duplicate. I documented the property in https://github.com/remotestorage/remotestorage.js/pull/1037/files#diff-a18ce18f9db57ab2aeeb4b2e649342eaL184

Reply:
Cool.